app/models/ems_refresh/save_inventory.rb

```diff
@@ -391,4 +391,12 @@ def save_ems_storage_inventory(ems, hashes, target = nil)
     save_ems_block_storage_inventory(ems, hashes, target) if ems.supports?(:block_storage)
     save_ems_object_storage_inventory(ems, hashes, target) if ems.supports?(:object_storage)
   end
+
+  #
+  # Saves asset details information of a resource
+  #
+  def save_asset_details_inventory(parent, hash)
```

Comment:
@walteraa can you move this method to [`save_inventory_physical_infra`](https://github.com/lenovo/manageiq/blob/93e339c453b3d09b74bf08812f63a6fb70bcfd0f/app/models/ems_refresh/save_inventory_physical_infra.rb)?

The base `save_inventory.rb` is more for general purpose inventory methods.